### ISSUE TITLE
Firebase offline first

### ISSUE THREAD

**Issue by calebeaires** [yeldarby/offline-firebase#3]
Does this js works with the new firebase version?

**Comment by mcdonamp** [comment]
Two potential issues: First, the need for a debug version (which is included with every release, just append `-debug` to the file before downloading.

Second is that v0 is long deprecated, it's pinned to v1.0.6 of our JS client and we're currently at [2.3.2](https://cdn.firebase.com/js/client/2.3.2/firebase-debug.js). I would try this with the latest version (debug or minified) to see if it works.

**Comment by yeldarby** [comment]
Thanks, I didn't know where to find the latest version. I will have a shot at updating this repo to use the more recent API

We're using this in production; any timeframe for v0 ceasing to work?

**Comment by mcdonamp** [comment]
Yep, no problem. I basically just try to get people to upgrade from v0 since we've implemented a ton of new features, performance improvements, and bug fixes since then. No immediate plans for deprecation (we try to always implement backwards compatible changes), but I would try to upgrade for the performance and bug fixes if for no other reason :)

**Comment by davidLeonardi** [comment]
+1 for an update on this repo!

**Comment by matt-webfab** [comment]
Also +1 for an update.
